### PR TITLE
Feature/default image dir

### PR DESCRIPTION
**Pull request by beccadawson**
Ticket: https://webjobs.agepartnership.co.uk/desk/tickets/6599885
See https://agepartnershipweb.slack.com/archives/C59CNHY2J/p1676020120412159 for detail of issue

Checks:
- Standard ISO sanity & malice checks
- Implementation ok

Preview:
- See https://github.com/AgePartnership/new.web.agepartnership.co.uk/pull/21 for implementation on new site
- See https://github.com/AgePartnership/oleg-component-library/pull/63 for implementation on component library
- See https://github.com/AgePartnership/web.agepartnership.co.uk/pull/3929 for demo of change not breaking legacy site

### CODE REVIEW

**Comment by beccadawson** [comment]
@ryanfitton-ap have added preview for icons functioning on existing site – they don't use the icons partial so aren't actually impacted by the amends

Also added @marymooney review as PM for existing site rebuild

File: rebuild/_config.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+$brand-assets-dir: '../assets/' !default;
```
Comment (ryanfitton-ap) [inline rebuild/_config.scss:1]: May need to remove the trailing slash here, or remove the first slash `rebuild/config/_icons.scss` at `'/images/...`.

If this is compiled as-is, the CSS will have a double slash e.g.

```
background-image:url(../assets//images/icons/hamburger-close--primary.svg)
```

**Review by ryanfitton-ap — CHANGES_REQUESTED** [review]
Can you also update some text in the `test-icons.html` file. This file was used to display all icons together.
 If you can, update to:
 
```
        <!-- Notes - test in browser -->
        <p>Update the `$brand-assets-dir` variable in `rebuld/_config.scss` to</p>
        <p><code>$brand-assets-dir: 'rebuild/assets' !default;</code></p>
        <p>Compile the SASS files:</p>
        <p><code>sass --watch 'rebuild/_config.scss':'test-icons.css' --style compressed</code></p>
        <p>And open the test-icons.html file in your browser</p>
```

File: rebuild/_config.scss (hunk 1 above)
Comment (beccadawson) [reply rebuild/_config.scss:1]: Have removed slash from paths in _icons

**Comment by beccadawson** [comment]
@ryanfitton-ap have updated demo instructions to use brand-assets-var and have added PR to update implementation on component library

**Review by ryanfitton-ap — APPROVED** [review]

File: test-icons.html [hunk 2]
```diff
@@ -9,11 +9,9 @@
 
     <body>
         <!-- Notes - test in browser -->
-        <p>Create a symlink for the icon asset locations:</p>
-        <p><code>mkdir node_modules/oleg-brand-kit && ln -s ../../rebuild node_modules/oleg-brand-kit/rebuild</code></p>
-        <p>Compile the SASS files:</p>
-        <p><code>sass --watch 'rebuild/_config.scss':'test-icons.css' --style compressed</code></p>
-        <p>And open the test-icons.html file in your browser</p>
+        <h2>Preview Instructions</h2>
+        <p><code>sass --watch 'test-icons.scss':test-icons.css --style compressed</code></p>
```
Comment (kimoshinowo98) [inline test-icons.html:13]: Am I supposed to be able to run this command? I get `zsh: command not found: sass`

**Review by kimoshinowo98 — CHANGES_REQUESTED** [review]

File: test-icons.html (hunk 2 above)
Comment (beccadawson) [reply test-icons.html:13]: Just means you don't already have sass installed – `npm install -g sass`

**Review by kimoshinowo98 — APPROVED** [review]

**Pull request merged**